basic_bootloader/src/bootloader/process_transaction.rs

```diff
@@ -485,6 +485,12 @@ where
 
         let chain_id = system.get_chain_id();
 
+        // Process access list
+        // Note: this operation should be performed before the hashing of the
+        // transaction, as the latter assumes the transaction structure has
+        // already been validated.
+        transaction.process_access_list(system, &mut resources)?;
```

Comment:
nit: it is not very obvious what "process access list" means

Reply:
Renamed to something more meaningful